source/quick-start.txt

```diff
@@ -11,215 +11,235 @@ Quick Start
-<http://pypi.python.org/pypi/pymongo/>`_.
+**PyMongo** is in the `Python Package Index.
+<http://pypi.python.org/pypi/pymongo/>`__
 
-.. warning:: **Do not install the "bson" package from pypi.** PyMongo comes
-   with its own bson package; doing "pip install bson" or "easy_install bson"
+.. warning::
+  
+   Do not install the ``bson`` package from PyPI. PyMongo comes
+   with its own ``bson`` package. Running ``pip install bson`` or ``easy_install bson``
    installs a third-party package that is incompatible with PyMongo.
 
 Installing with pip
 -------------------
 
-We recommend using `pip <http://pypi.python.org/pypi/pip>`_
-to install pymongo on all platforms:
+We recommend using `pip <http://pypi.python.org/pypi/pip>`__
+to install PyMongo on all platforms:
 
 .. code-block:: python
 
-  $ python3 -m pip install pymongo
+   $ python3 -m pip install pymongo
 
-To get a specific version of pymongo:
+To get a specific version of PyMongo:
 
 .. code-block:: python
 
-  $ python3 -m pip install pymongo==3.5.1
+   $ python3 -m pip install pymongo==3.5.1
 
-To upgrade using pip:
+To upgrade by using pip:
 
 .. code-block:: python
 
-  $ python3 -m pip install --upgrade pymongo
+   $ python3 -m pip install --upgrade pymongo
 
 Dependencies
 ------------
 
 PyMongo supports CPython 3.7+ and PyPy3.7+.
 
-Required dependencies
-.....................
-
-Support for mongodb+srv:// URIs requires `dnspython
-<https://pypi.python.org/pypi/dnspython>`_
+Support for mongodb+srv:// URIs requires `dnspython.
+<https://pypi.python.org/pypi/dnspython>`__
 
-Installing from source
+Installing from Source
 ----------------------
 
-If you'd rather install directly from the source (i.e. to stay on the
-bleeding edge), install the C extension dependencies then check out the
+If you'd rather install PyMongo directly from the source,
+install the C-extension dependencies, and then check out the
 latest source from GitHub and install the driver from the resulting tree:
 
 .. code-block:: python
 
-  $ git clone https://github.com/mongodb/mongo-python-driver.git pymongo
-  $ cd pymongo/
-  $ pip install .
+   $ git clone https://github.com/mongodb/mongo-python-driver.git pymongo
+   $ cd pymongo/
+   $ pip install .
 
-Installing from source on Unix
-..............................
+Installing from Source on Unix
+``````````````````````````````
 
-To build the optional C extensions on Linux or another non-macOS Unix you must
-have the GNU C compiler (gcc) installed. Depending on your flavor of Unix
-(or Linux distribution) you may also need a python development package that
+To build the optional C extensions on Linux or another non-macOS Unix, you must
+have the GNU C compiler (``gcc``) installed. Depending on your flavor of Unix
+(or Linux distribution), you may also need a Python development package that
 provides the necessary header files for your version of Python. The package
-name may vary from distro to distro.
+name might vary from distro to distro.
```

Comment:
```suggestion
name might vary between OS versions.
```

I don't know if we should use "operating system" instead of OS?

Reply:
To me, "OS versions" suggests different numbered versions of the same OS, whereas "distro" means a different flavor of linux. I'll reword for clarity though